src/lib/components/event/event-details-row.svelte

```diff
@@ -35,7 +35,7 @@
       {format(key)}
     </p>
     <CodeBlock content={getCodeBlockValue(value)} class="w-[95%]" {inline} />
-  {:else if shouldDisplayAsExecutionLink(key)}
+  {:else if shouldDisplayAsExecutionLink(key) && workflow && namespace}
```

Comment:
Maybe we need a new utility at this point? Or maybe `shouldDisplayAsExecutionLink` needs more optional arguments?